include/radio.hpp

```diff
@@ -145,6 +156,17 @@ namespace bzzz
          */
         void waitForArmCommand();
 
+        /**
+         * @brief send IMU data quaternions and acclerometer values to Pi
+         * @param q1
+         * @param q2
+         * @param q3
+         * @param ax
+         * @param ay
+         * @param az
+        */
+        void sendIMUDataToPi(float q1, float q2, float q3, float ax, float ay, float az, float motorFL, float motorFR, float motorBL, float motorBR);
```

Comment:
This method is called `sendIMUDataToPi` but it also sends the motor commands? Are the motor commands needed to Pi?

Reply:
Renamed the function `sendIMUDataToPi` to `sendFlightDataToPi`. Currently, we need the motor data for plotting purposes.